Added new command cybereason-update-malop-investigation-status to Cybereason integration

Stjubit opened the pull request:
## Contributing to Cortex XSOAR Content

Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues

None

## Description

If a Cybereason customer uses the new data infrastructure in their environment, the MalOp status has to be updated using a new API and the status is now called "Investigation Status".

This PR adds a new command `cybereason-update-malop-investigation-status` to the Cybereason integration.

## Must have

- [X] Tests
- [X] Documentation

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/demisto/content?pullRequest=37296) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/demisto/content?pullRequest=37296) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/demisto/content?pullRequest=37296) it.</sub>

Stjubit commented:
Ready for Review!

samuelFain commented:
@Stjubit we appreciate your patience. We are still waiting for the partner's approval for this PR. We attempted to reach them again, and we will update the PR's status as soon as possible. Thank you!

MLainer1 commented:
Hi @Stjubit, we got approval from the Partner to continue with your PR.
The review will be posted shortly. In the meantime, make sure to fix the conflicts and the pre-commit errors.
Thanks

MLainer1 commented on Packs/Cybereason/Integrations/Cybereason/Cybereason.py:
```diff
@@ -2011,6 +2020,35 @@ def cybereason_process_attack_tree_command(client: Client, args: dict):
+
+    response = client.cybereason_api_call('PUT', f'/rest/mmng/v2/malops/{malop_guid}', json_body=json_body)
+    if response['status'] != 'SUCCESS':
+        raise Exception(
```
```suggestion
        raise DemistoException(
```

MLainer1 commented on Packs/Cybereason/Integrations/Cybereason/Cybereason.py:
```diff
@@ -2011,6 +2020,35 @@ def cybereason_process_attack_tree_command(client: Client, args: dict):
 
 
+def update_malop_investigation_status_command(client: Client, args: dict):
+    malop_guid = str(args.get('malopGuid'))
+    investigation_status = str(args.get('investigationStatus'))
```
```suggestion
    malop_guid = args.get('malopGuid')
    investigation_status = args.get('investigationStatus')
```
No need to use the str() method, as it is already a string

Stjubit replied:
If I don't tag it as string, linting will fail. All other commands also do it like that.

MLainer1 reviewed (changes requested):
Thank you for your contribution!
Let's schedule a short demo to check your changes. The demo will be accompanied by someone from Cybereason as well, I'll contact you through an email.

MLainer1 commented:
Hi @Stjubit, the partner will be available for a demo after 2.1.2025 .
I'll reach over via email to schedule the demo.

MLainer1 commented:
Hi @Stjubit, I've sent an email about scheduling a demo for this week

MLainer1 commented:
Hi @Stjubit, just to update you:
Sorry for the delay. We have been trying to reach Cybereason for the demo for some time. If they do not respond in the next couple of days, we will probably close this PR until they do.

MLainer1 commented:
Hi @Stjubit, the partner got back to me, their availability is as follows:
Feb 4: 11 AM - 3 PM, 5 PM - 7 PM
Feb 5: 12 PM - 2 PM, 5 PM - 7 PM

All the above times are in Indian Standard Time (IST)

Let me know if it works for you

Stjubit commented:
Hi,

sorry I was out-of-office last week.

Is it really necessary to do a meeting with Cybereason? I mean I just added a simple command. 👀 

I made the branch up-to-date with master and implemented your feedback from the code review.

Somehow I can't pass the CLA check. I signed the CLA with both accounts in this PR's commits (`Stjubit` and `nts-jkaufmann`). Do you know how I can fix this?

MLainer1 commented:
Hi @Stjubit, Cybreason requested the demo from their end. What is your availability for this week?

Stjubit commented:
hmmm okay. That sucks a little bit to be honest.

**My availabilities next week:**

- Wednesday, 12.02.2025, 09.00 - 16.00 CET (GMT+1)
- Thursday, 13.02.2025, 09.00 - 17.00 CET (GMT+1)

MLainer1 commented:
Hi @Stjubit, I've sent an invitation for 13.02 09.00 CET

MLainer1 commented:
Hi @Stjubit , I saw this line in the CLA comment:
`Stjubit seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).`

Can you try to follow the steps there?
